server/utils/excelExport/poamExport.ts

```diff
@@ -374,7 +378,31 @@ export async function generatePoam(
   const recommendations = 20;
 
   const results = await getEvaluationSummary(boundaryId, undefined, true);
-  const boundary = await Boundary.findByPk(boundaryId);
+
+  interface BoundaryWithClassification extends Boundary {
```

Comment:
usually one should not define an interface in-line in a function

after further discussion, yall are in the process of phasing these out because the sequelize models now include the join table info so you don't need to do it manually here.

there are other locations where this is happening